### DISCUSSION TITLE
授权设备使用专业版失败！

### DISCUSSION THREAD

**Discussion by LZheng1103** [getlantern/lantern-forum-cn#996] [General]
在mac上装的专业版（从官网上下载的新版本），授权其他设备（window）时显示已经授权成功，但被授权的设备还是免费版！我未在其他设备上授权过！前面几年用旧版本时都很顺利授权。请问怎么解决？

**Comment by LZheng1103** [comment]
另外，我现在时专业版，想请问如何要到激活码？

**Comment by LZheng1103** [comment]
![15841714009662_ pic](https://github.com/user-attachments/assets/fb92268c-eeee-4d3d-a2eb-506534cc57c1)
这是我的续费记录

**Comment by LZheng1103** [comment]
我邮箱：14459446@qq.com

**Comment by MaxMKMax** [comment]
@LZheng1103 您好. 已经帮您提交上去了. 过几天会帮您弄好.

**Comment by LZheng1103** [comment]
谢谢！


LZheng




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月26日(星期六) 晚上11:05
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 授权设备使用专业版失败！ (Discussion #996)





 
@LZheng1103 您好. 已经帮您提交上去了. 过几天会帮您弄好.

**Comment by LZheng1103** [comment]
你好，请问弄好了吗


LZheng




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月26日(星期六) 晚上11:05
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 授权设备使用专业版失败！ (Discussion #996)





 
@LZheng1103 您好. 已经帮您提交上去了. 过几天会帮您弄好.

**Comment by MaxMKMax** [comment]
请您给我们多几天. 我这里还等待回复呢. 不好意思啊.

**Comment by LZheng1103** [comment]
好的，谢谢


LZheng




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年11月5日(星期二) 晚上11:33
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 授权设备使用专业版失败！ (Discussion #996)





 
请您给我们多几天. 我这里还等待回复呢. 不好意思啊.
